conf/app.routes

```diff
@@ -1,4 +1,6 @@
 # microservice specific routes
 
-GET         /scratch/:id        controllers.ScratchController.get(id)
-POST        /scratch            controllers.ScratchController.save
\ No newline at end of file
+GET         /scratch/:id          controllers.ScratchController.get(id)
+POST        /scratch              controllers.ScratchController.save
+
+GET         /publish/process/:id  controllers.PublishedController.get(id)
```

Comment:
I may have missed the discussion here, but why have you added "process" into the route here for publish?